apps/extension/package.json

```diff
@@ -6,13 +6,16 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc -b && vite build",
+    "watch": "concurrently \"tsc -b -w\" \"vite build --watch\"",
```

Comment:
This is just for convenience really, so I could have a watch script running in the BG while I code the extension and have it show up real-time. Installed a package Concurrently globally to make use of it.

Reply:
hehe i yes i've used in the past